library/CodeMonkeysRu/GCM/Exception.php

```diff
@@ -9,5 +9,31 @@ class Exception extends \Exception
     const MALFORMED_REQUEST = 3;
     const UNKNOWN_ERROR = 4;
     const MALFORMED_RESPONSE = 5;
+    const INTERNAL_SERVER_ERROR = 6;
+    const SERVICE_UNAVAILABLE = 7;
```

Comment:
Agreed, removing. SERVICE_UNAVAILABLE seems more serious and we might want to shut off our push notifications for a while in that case. But someone can always just check the response headers now if they want to implement different handling.